commands/verification/verify.js

```diff
@@ -71,7 +71,7 @@ module.exports = class Verification extends PermissionCommand {
                     .addField('Don\'t Forget!', 'Remember you need to !attend <your email> in the attend channel that will open a few hours before the hackathon begins.');
                 discordServices.replaceRoleToMember(member, discordServices.roleIDs.guestRole, discordServices.roleIDs.hackerRole);
                 discordServices.addRoleToMember(member, discordServices.roleIDs.memberRole);
-                if (discordServices.stampRoles.has(0)) discordServices.addRoleToMember(member,discordServices.stampRoles.get(0));
+                if (discordServices.stampRoles.has(0)) discordServices.addRoleToMember(member,discordServices.stampRoles.find(number => number === 0));
```

Comment:
Could you add a TODO comment here just so that I can find it thanks @mwang2000

Reply:
Added; also included a reminder to change the same thing in classes/verification.js since it still operates on the old structure of stampRoles that is Collection<stampNumber, roleID>.